### ISSUE TITLE
Auto generated migration doesnt valid for PSR12

### ISSUE THREAD

**Issue by kirya-dev** [doctrine/migrations#1119]
### Bug Report

Auto generated migration doesnt valid for PSR12

```
FILE: /*****/migrations/Version20210129094711.php
------------------------------------------------------------------------------------------
FOUND 3 ERRORS AFFECTING 3 LINES
------------------------------------------------------------------------------------------
 15 | ERROR | [x] There must not be a space before the colon in a return type declaration
 20 | ERROR | [x] There must not be a space before the colon in a return type declaration
 28 | ERROR | [x] There must not be a space before the colon in a return type declaration
------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 3 MARKED SNIFF VIOLATIONS AUTOMATICALLY
------------------------------------------------------------------------------------------

Time: 8.3 secs; Memory: 36MB
```

**Comment by goetas** [comment]
You can provide a custom template, that follows any coding standard you want. See https://github.com/doctrine/migrations/blob/780b67ec03039b3a4f2777614666d480137efebe/lib/Doctrine/Migrations/Configuration/Configuration.php#L36

**Comment by kirya-dev** [comment]
> You can provide a custom template

No. Target of this PR will change default behavior.

**Comment by guilliamxavier** [comment]
Related: https://github.com/doctrine/coding-standard/pull/163 and https://github.com/doctrine/migrations/commit/ef842cfb1944606148159e724d8ae7e7845cf4b0

**Comment by jtrupina** [comment]
@kirya-dev I also came across this. Any new updates on this issue? Also there is already open PR: https://github.com/doctrine/migrations/pull/875 related to this issue. Also, @guilliamxavier I don't see that https://github.com/doctrine/coding-standard/pull/163 fixed this particular issue or am I missing something.

**Comment by guilliamxavier** [comment]
@jtrupina: I didn't say "fixed"; to be clear: https://github.com/doctrine/coding-standard/pull/163 updated the Doctrine CS rule for return type declarations from "1 space before colon" (historical Doctrine style) to "no space before colon" (PSR-12), and later #1102 (more precisely https://github.com/doctrine/migrations/pull/1102/commits/53a5da0a0b1ca5cc90061b128c4bed956e6ae754 and https://github.com/doctrine/migrations/pull/1102/commits/ef842cfb1944606148159e724d8ae7e7845cf4b0) upgraded and applied the new CS to this lib (code), but left out the template (string). Thanks for mentioning #875, you're right 🙂 the thing is it was created before https://github.com/doctrine/coding-standard/pull/163 so was put on hold, and seems to have been forgotten afterwards... I'll ping it 😉

**Comment by greg0ire** [comment]
I think we can close this, it will be fixed in the next 3.1.x release :)
